bundle/edu.gemini.util.security/build.sbt

```diff
@@ -11,24 +11,18 @@ unmanagedJars in Compile ++= Seq(
 )
 
 libraryDependencies ++= Seq(
-  "org.scala-lang.modules" %% "scala-swing" % "2.0.0-M2",
-  "org.scalaz.stream"      %% "scalaz-stream" % "0.8a" intransitive(),
   "org.scalaz"             %% "scalaz-core" % ScalaZVersion,
   "org.scalaz"             %% "scalaz-effect" % ScalaZVersion,
   "org.scalaz"             %% "scalaz-concurrent" % ScalaZVersion,
-  "org.tpolecat"           %% "doobie-core" % "0.3.0-M1",
-  "com.chuusai"            %% "shapeless" % "2.3.0",
-  "org.scodec"             %% "scodec-bits" % "1.0.9",
-  "org.scala-lang"         %  "scala-reflect" % "2.11.7",
-  "org.scala-lang"         %  "scala-compiler" % "2.11.7"
```

Comment:
They come as dependencies from doobie, thus we don't need to keep the versions in sync